bartop-ui/src/components/Auth/Auth.js

```diff
@@ -1,132 +0,0 @@
-import React, { Component } from 'react';
-import PropTypes from 'prop-types';
-import { connect } from 'react-redux';
-import { Switch, Route } from 'react-router-dom';
-
-import PageWithHeader from '../Layouts/PageWithHeader/PageWithHeader';
-import LoginForm from './LoginForm/LoginForm';
-import VerifyForm from './VerifyForm/VerifyForm';
-import Callback from './Callback/Callback';
-import Failure from './Failure/Failure';
-import { actions } from '../../ducks/authentication/authentication';
-import NotFound from '../NotFound/NotFound';
-
-export class Auth extends Component {
```

Comment:
The Auth flow is now in modals, instead of different routes, so all the `/auth/` and related components are no mas